CHANGES.md

```diff
@@ -5,6 +5,7 @@ Features ✨:
  -
 
 Improvements 🙌:
+ - Full compatibility with homeservers on Tor (#2941)
```

Comment:
Can you change that to something more related to the actual change like `Allow Element Android to connect to homeservers on Tor` for instance?

Reply:
Changed it to "Allow non-HTTPS connections to homeservers on Tor". Does that work?